src/orbitdetermination/neosolution.jl

```diff
@@ -12,7 +12,7 @@ The outcome of the orbit determination process for a NEO.
 - `g_bwd/g_fwd::Vector{U}`: geocentric distance at close approach.
 - `res::Vector{OpticalResidual{T, U}}`: vector of optical residuals.
 - `fit::LeastSquaresFit{T}`: least squares fit.
-- `scalings::Vector{T}`: jet transport scaling factors.
+- `jacobian::Matrix{T}`: residuals space to barycentric coordinates jacobian.
```

Comment:
`jacobian` is a generalization of `scalings`. The former reduces to the latter when a `TaylorN` takes the form `x + dx`. Otherwise, for a nontrivial JT tail, `jacobian` is the full projection of the covariance matrix from residuals to barycentric space.